tests/unit/test_config.py

```diff
@@ -46,11 +48,22 @@ def config_dir(file_config_vals):
         yield dirname
 
 
+@pytest.fixture
+def bad_config_dir(file_config_vals):
+    with tempfile.TemporaryDirectory() as dirname:
+        fullpath = os.path.join(dirname, "config.yaml")
+        with open(fullpath, "w+") as f:
+            f.write("this\nis: a bad\t\n\t config file::::")
+
+        yield dirname
```

Comment:
Tests like this one weren't really part of this PR's logic, but they were to improve the coverage. For any additional tests I'd rather make a new PR.